doc/en/api/1.0.0/resource.yaml

```diff
@@ -0,0 +1,210 @@
+---
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: GeoServer Resources
+  description: A resource is any item in the data directory that does not represent configuration. Typical resources include styles and icons.
+
+paths:
+  /resource/{pathToResource}:
+    parameters:
+      - name: pathToResource
+        in: path
+        type: string
+        required: true
+        description: The full path to the resource. Required, but may be empty; a request to `resource` references the top level resource directory.
```

Comment:
If it is empty does that make `required: true` above incorrect? Or is this a complicated way of saying that the `/` is required....

Reply:
The slash isn't required, no (its not part of the path parameter anyways).
I considered required=false, that just seemed a bit weird for a path parameter. If you think that would be clearer, I am happy to change it.

Reply:
Just tried out with required=false, and apparently the swagger rules don't like it if you have a path variable that isn't required, so I guess that will have to stay as-is